share/wake/lib/system/environment.wake

```diff
@@ -53,6 +53,12 @@ publish path = match (getenv "WAKE_PATH")
 publish path = "/usr/bin", Nil
 publish path = "/bin", Nil
 
+# On MacOS and FreeBSD, many important system binaries are not in /usr/bin
+publish path = match sysname
+    "Darwin" = "/opt/local/bin", "/usr/local/bin", Nil
+    "FreeBSD" = "/usr/local/bin", Nil
```

Comment:
What motivates this change?

Reply:
These were in os.wake previously.

Reply:
Admittedly not a lot.  That block was the *only* code in `os.wake`, and the blocks above it are the only other ones that publish to `path`, so it's just a bit of "bring everything related to the `topic` into one place".  If we have more OS-dependent code, keeping the separate file would probably make a bit more sense, but as is this seemed like it had a chance of getting lost.